### PR TITLE
more tests for deleting funding source

### PR DESCRIPTION
**Pull request by anissa-agahchen**

### CODE REVIEW

File: api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts [hunk 1]
```diff
@@ -97,4 +110,52 @@ describe('delete a funding source', () => {
       expect(actualError.message).to.equal('Failed to build SQL delete statement');
     }
   });
+
+  it('should throw a 400 error when the delete fundingSource fails, because the response has no rows', async () => {
+    const mockQuery = sinon.stub();
+
+    mockQuery.onFirstCall().resolves({ rows: [{ id: 1 }] });
```
Comment (sdevalapurkar) [inline api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts]:
```suggestion
    mockQuery.resolves({ rows: [{ id: 1 }] });
```

File: api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts [hunk 2]
```diff
@@ -97,4 +110,52 @@ describe('delete a funding source', () => {
...
+    } catch (actualError) {
+      expect(actualError.status).to.equal(400);
+      expect(actualError.message).to.equal('Failed to delete project funding source');
+    }
+  });
+
+  it('should return the row count of the removed funding source on success', async () => {
+    const mockQuery = sinon.stub();
+
+    mockQuery.onFirstCall().resolves({ rowCount: 1 });
```
Comment (sdevalapurkar) [inline api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts]:
```suggestion
    mockQuery.resolves({ rowCount: 1 });
```

File: api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts [hunk 3]
```diff
@@ -97,4 +110,52 @@ describe('delete a funding source', () => {
       expect(actualError.message).to.equal('Failed to build SQL delete statement');
     }
   });
+
+  it('should throw a 400 error when the delete fundingSource fails, because the response has no rows', async () => {
```
Comment (sdevalapurkar) [inline api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts]: why is this test failing? it says when response has no rows but you are returning a non null rows value

**Review by sdevalapurkar — CHANGES_REQUESTED** [review]

File: api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts (hunk 1 above)
Comment (anissa-agahchen) [reply api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts]: right .. makes sense

**Review by sdevalapurkar — APPROVED** [review]

**Pull request merged**